Formula/yamale.rb

```diff
@@ -0,0 +1,52 @@
+class Yamale < Formula
+  include Language::Python::Virtualenv
+
+  desc "Schema and validator for YAML"
+  homepage "https://github.com/23andMe/Yamale"
+  url "https://files.pythonhosted.org/packages/c9/7b/3c0e9ecf5f5ad25700cc9993efb607d748658b2cd2492bda0adf8b6087dc/yamale-4.0.2.tar.gz"
+  sha256 "4168f8b3650cece80552fd32edd894ab9081dd9ef959cadd9f1f23795629e4f2"
+  license "MIT"
+  revision 1
```

Comment:
```suggestion
```
No need for a revision

Reply:
Will drop, is there a reason that `brew audit --new` didn't scream at me about this?

Reply:
Because we're not checking for that. There isn't really a reason to add a revision to a new formula so we've never automated the check for it.

Reply:
If I tried to write a PR to add a check for it, would it be acceptable?

I modeled this formula off an existing formula which is why the field was there, and it didn't seem obviously wrong and nothing screamed at me. (When I left an empty `bottle` section, robocop complained.)

Reply:
Yeah, that makes sense to me. Let me know when you have a PR and I'll take a look.

Reply:
homebrew/brew#12854